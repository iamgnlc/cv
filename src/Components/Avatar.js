import React, { PureComponent } from "react";

import { Media } from "reactstrap";

import avatar from "../Images/avatar.jpg";

class Avatar extends PureComponent {
  render() {
    return <Media object src={avatar} className="avatar rotate mr-2" />;
  }
}

export default Avatar;
