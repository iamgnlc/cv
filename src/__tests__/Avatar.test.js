import React from "react"
import renderer from "react-test-renderer"

import Avatar from "../Components/Avatar"

describe("avatar component", () => {
  it("should render without crashing", () => {
    const tree = renderer.create(<Avatar />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
