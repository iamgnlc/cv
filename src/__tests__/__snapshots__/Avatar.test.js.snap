// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`avatar component should render without crashing 1`] = `
<img
  className="avatar rotate mr-2 media-object"
  src="avatar.jpg"
/>
`;
